### PR TITLE
[GEN-160] Image upload button in write (climbList)

### PR DESCRIPTION
**Pull request by redpepper-nag**
The button leads is not functional as it does not have an onPress handler (which is my next PR)
IMG_9481

### CODE REVIEW

File: trk/src/Screens/ClimbList/index.js [hunk 1]
```diff
@@ -93,12 +93,11 @@ const ClimbInputData = () => {
         />
 
         <Text style={styles.label}>Image</Text>
-        <TextInput
-          style={styles.input}
-          value={image}
```
Comment (redpepper-nag) [inline trk/src/Screens/ClimbList/index.js:98]: removed placeholder image form

File: trk/src/Screens/ClimbList/index.js [hunk 2]
```diff
@@ -93,12 +93,11 @@ const ClimbInputData = () => {
...
-          onChangeText={setImage}
-          placeholder="Enter image"
-        />
```
Comment (redpepper-nag) [inline trk/src/Screens/ClimbList/index.js:101]: removed placeholder image form

File: trk/src/Screens/ClimbList/index.js [hunk 3]
```diff
@@ -1,5 +1,5 @@
 import React, { useState, useEffect, useContext } from "react";
-import { SafeAreaView, View, Text, StyleSheet, TextInput, Image, Button, Alert } from "react-native";
+import { SafeAreaView, View, Text, StyleSheet, TextInput, Image, Button, Alert, TouchableOpacity } from "react-native";
```
Comment (redpepper-nag) [inline trk/src/Screens/ClimbList/index.js:2]: imported TouchableOpacity (button component)

File: trk/src/Screens/ClimbList/index.js [hunk 4]
```diff
@@ -108,21 +107,13 @@ const ClimbInputData = () => {
       </View>
 
       <Button
-        style={styles.button}
```
Comment (redpepper-nag) [inline trk/src/Screens/ClimbList/index.js:111]: removed button styling which didnt exist in code

File: trk/src/Screens/ClimbList/index.js [hunk 5]
```diff
@@ -108,21 +107,13 @@ const ClimbInputData = () => {
...
       >
       </Button>
 
-      <Button
-        style={styles.button}
-        onPress={() => navigation.navigate("List")}
-        mode="contained"
-        title="Go to Climb List"
-      >
-      </Button>
```
Comment (redpepper-nag) [inline trk/src/Screens/ClimbList/index.js:125]: removed button that caused bug as it led to nowhere

File: trk/assets/image-icon.png
Comment (redpepper-nag) [inline trk/assets/image-icon.png:1]: imported image icon used in touchable opacity button

**Review by VGmt — APPROVED** [review]
Looks perfect

**Pull request merged**